samples/WorkerBindingSamples/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Hosting;
+
+namespace SampleApp
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            Console.WriteLine($"Azure Functions .NET Worker (PID: { Environment.ProcessId }) initialized in debug mode. Waiting for debugger to attach...");
```

Comment:
Do we need this console.writeline message? The IDE is not going to wait here. rite ?

Reply:
I use it for debugging, I can attach to the PID without having the IDE wait here but we don't need to include it in the sample